ui/restHandler.go

```diff
@@ -313,9 +325,49 @@ func Run(w http.ResponseWriter, r *http.Request, jd map[string]interface{},
 // RESTStatusHandler will print the state of the runs.
 func RESTStatusHandler(w http.ResponseWriter, r *http.Request) {
 	fhttp.LogRequest(r, "REST Status Api call")
+
+	runid, _ := strconv.ParseInt(r.FormValue("runid"), 10, 64)
+	statusResult, err := StatusByRunID(runid)
+
 	w.Header().Set("Content-Type", "application/json")
-	w.WriteHeader(http.StatusServiceUnavailable)
-	w.Write([]byte("{\"error\":\"status not yet implemented\"}"))
+
+	if err != nil {
+		w.WriteHeader(http.StatusNotFound)
+		w.Write([]byte(fmt.Sprintf("{\"error\": %s}", err.Error())))
+	} else {
```

Comment:
return in error if, no else (never have an else is a good rule of thumb, unless both if and else are 1 line each or so)